media-autobuild_suite.bat

```diff
@@ -1900,7 +1912,7 @@ set compileArgs=--cpuCount=%cpuCount% --build32=%build32% --build64=%build64% ^
     if %build64%==yes (
         if %CC%==clang ( set "MSYSTEM=CLANG64" ) else set "MSYSTEM=MINGW64"
     ) else (
-        if %CC%==clang ( set "MSYSTEM=CLANG32" ) else set "MSYSTEM=MINGW32"
+        if %CC%==gcc set "MSYSTEM=MINGW32"
     )
```

Comment:
Same here regarding `CC==clang && build32=yes` being an invalid state, so assuming `build64 != yes`, then `build32` must equal yes, and thus `CC` must be `gcc` (at least in our case).